zappnode-front/src/routes/Dashboard.js

```diff
@@ -1,179 +1,21 @@
-            </div>
-          </div>
-        </div>
-        <Button
-          onClick={(e) => {
-            setSelectedTracker(tracker);
-            setVisible(true);
-          }}
-        >
-          Search
-        </Button>
-      </div>
-    );
-  };
-
-  const renderGridItem = (tracker) => {
-    return (
-      <div style={{ padding: '.5em' }} className="p-col-12 p-md-3">
-        <Panel header={tracker.name} style={{ textAlign: 'center' }}>
-          <div>{tracker.name}</div>
-          <Button
-            onClick={(e) => {
-              setSelectedTracker(tracker);
-              setVisible(true);
-            }}
-          >
-            Search
-          </Button>
-        </Panel>
-      </div>
-    );
-  };
-
-  const itemTemplate = (tracker, layout) => {
-    if (!tracker) {
-      return;
-    }
-    if (layout === 'list') return renderListItem(tracker);
-    else if (layout === 'grid') return renderGridItem(tracker);
-  };
-
-  const renderTrackerDialogContent = () => {
-    if (selectedTracker) {
-      return (
-        <div className="p-grid" style={{ fontSize: '16px', textAlign: 'center', padding: '20px' }}>
-          <div className="p-col-4">Percentage: </div>
-          <div className="p-col-8">{selectedTracker.percentage}</div>
-
-          <div className="p-col-4">Name: </div>
-          <div className="p-col-8">{selectedTracker.name}</div>
-
-          <div className="p-col-4">Type: </div>
-          <div className="p-col-8">{selectedTracker.type}</div>
-        </div>
-      );
-    } else {
-      return <div>Empty</div>;
-    }
-  };
-
-  const renderHeader = () => {
-    const sortOptions = [{ label: 'A-Z', value: 'name' }];
-
-    return (
-      <div className="p-grid">
-        <div className="p-col-6" style={{ textAlign: 'left' }}>
-          <Dropdown
-            options={sortOptions}
-            value={sortKey}
-            placeholder="Sort By"
-            onChange={onSortChange}
-            style={{ width: '12em' }}
-          />
-        </div>
-        <div className="p-col-6" style={{ textAlign: 'right' }}>
-          <DataViewLayoutOptions layout={layout} onChange={(e) => setLayout(e.value)} />
-        </div>
-      </div>
-    );
-  };
-
-  const header = renderHeader();
   return (
-    <div className="mb-10 dataview-demo">
-      <DataView
-        value={trackers}
-        layout={layout}
-        header={header}
-        itemTemplate={itemTemplate}
-        paginatorPosition={'both'}
-        paginator={true}
-        rows={3} // Value must be lower than `trackers.length`
-        sortOrder={sortOrder}
-        sortField={sortField}
-      />
-
-      <Dialog header="Trackers" visible={visible} modal={true} onHide={() => setVisible(false)}>
-        {renderTrackerDialogContent()}
-      </Dialog>
+    <>
+      <div>
+        <DataView />
+      </div>
       <div className="mt-12">
         <DataTable />
       </div>
-    </div>
+      <div className="mt-12">
```

Comment:
Let's add some padding to these so they aren't sitting on the edges. 

```suggestion
      <div className="mt-12 p-5">
```